pkgs/development/perl-modules/generic/builder.sh

```diff
@@ -22,7 +22,11 @@ preConfigure() {
         fi
     done
 
-    perl Makefile.PL PREFIX=$out INSTALLDIRS=site $makeMakerFlags PERL=$(type -P perl) FULLPERL=\"$fullperl/bin/perl\"
+    if [ "$LINKTYPE" = "static" ]; then
```

Comment:
A small note would be apt, like

```suggestion
    # https://metacpan.org/pod/ExtUtils::MakeMaker#Static-Linking-of-a-new-Perl-Binary
    if [ "$LINKTYPE" = "static" ]; then
```